docs/pages/web/tiledata.js

```diff
@@ -0,0 +1,179 @@
+
+      <MainSection name="Best practices">
+        {/* <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use TileData paired with data visualization colors to make a visual connection between a chart and it’s related data"
+            defaultCode={`<TileData color="data-visualization-01" title="Blah" value="2" selected /> `}
+          />
+
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use colors outside of the Data Visualization palette. Please refer to the [Usage](/foundations/data_visualization/usage) and [Palette](/data_visualization/palette)."
+          />
+</MainSection.Subsection> */}
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Always present one tile in its selected state on default"
+            defaultCode={`<TileData color="01" title="Impressions" value="2M" selected trend={{ value: 5, accessibilityLabel: 'Trending up' }} /> `}
+          />
+
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use it to present a single option, since they don't need to be selected. Use a [Datapoint](/web/datapoint) instead."
+            defaultCode={`<TileData color="01" title="Impressions" value="2M" trend={{ value: 1, accessibilityLabel: 'Trending up' }}/> `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use TileData to switch between data on charts"
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use for page-level navigation, as TileData is to be used in conjunction with a chart. Use [Tabs](/web/tabs), [SegmentedControl](/web/segmentedcontrol), [SideNav](/web/sidenavigation), or another local navigation instead."
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <AccessibilitySection
+        name={generatedDocGen?.displayName}
+        description="Users should be able to navigate or activate a TagData using a keyboard or other input modalities. Be sure to include an `accessibilityLabel` for the screen reader within the child element."
+      />
+
+      <MainSection
+        name="Localization"
+        description="Make sure content that is placed in a data point is set-up to work in R-T-L languages. Be sure to localize text and `accessibilityLabel` in TileData and all subcomponents as well. When the text of the TileData reaches its Max width, either intentionally or through localization, will wrap as needed to display the full text. Keep this in mind when selecting wording for your TileData menu items. Note that localization can lengthen text by 20 to 30 percent. "
```

Comment:
nits:
- "…that is placed in _Datapoint_ is _set up_ to work in _RTL_ languages."
- "Be sure to localize `title`, `value`, and `trend.accessibilityLabel`."
- Which text is being referred to when wrapping? `title`? `value`? Something else? We need to be _very specific_ about these sorts of things.
- weird casing on "max width"